projects/pt1/python/test/compile_api/tracing.py

```diff
@@ -7,7 +7,7 @@
 
 import torch
 
-import torch_mlir
+import torch_mlir.torchscript as torchscript
```

Comment:
nit: you can do `from torch_mlir import torchscript` to avoid repeating `torchscript`